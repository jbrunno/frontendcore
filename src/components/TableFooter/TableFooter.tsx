import React from 'react';
import {
  TableFooter as MuiTableFooter,
  TableFooterProps as MuiTableFooterProps,
} from '@mui/material';

export type TableFooterProps = MuiTableFooterProps;

export function TableFooter(props: TableFooterProps) {
  return <MuiTableFooter {...props} />;
}
export default TableFooter;
