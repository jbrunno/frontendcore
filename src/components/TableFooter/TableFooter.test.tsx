import React from 'react';
import { render } from '@test-utils/component-testing';
import { TableFooter } from './TableFooter';

test('renders the TableFooter', () => {
  const { container } = render(<TableFooter />);
  expect(container).toMatchSnapshot();
});
