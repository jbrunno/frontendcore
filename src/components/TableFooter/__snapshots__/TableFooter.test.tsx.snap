// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the TableFooter 1`] = `
.emotion-0 {
  display: table-footer-group;
}

<div>
  <tfoot
    class="MuiTableFooter-root emotion-0"
  />
</div>
`;
